### ISSUE TITLE
Can't install @nestjs/cli

### ISSUE THREAD

**Issue by Musbell** [oven-sh/bun#545]
### Version

 v0.1.2

got stuck here:

```
bun install @nestjs/cli
bun install v0.1.2
  🚚 typescript [1338/1346]
  
  ```

**Comment by FinnRG** [comment]
@Musbell There are 2 possibilities:

1. If this issue only appears sometimes it is likely due to issues with the http client (#490)
2. If this happens consistently, you should be able to install each stuck dependency independently and resolve the issue (#438)

**Comment by lackovic** [comment]
This worked on Ubuntu 22.04 WSL:

```
$ bun i -g @nestjs/cli
bun add v0.1.3
 installed @nestjs/cli@9.0.0 with binaries:
  - nest
 245 packages installed [3.20s]

$ bun -version
0.1.3
```

I am not sure how to use nest CLI though:
```
$ nest new project-name
/usr/bin/env: ‘node’: No such file or directory
```
I suppose I should "tell" nest to use bun instead of node.

**Comment by FinnRG** [comment]
@lackovic Yes, it also works on my side. These install deadlocks are for some reason really specific to individual environments.
@Musbell Could you please test if this bug appears consistently and if it appears consistently, please try individually adding each "stuck" package as described in #438.

**Comment by Musbell** [comment]
@lackovic @FinnRG It works. I suppose this is specific to individual environments as @lackovic  stated above.

**Comment by Jarred-Sumner** [comment]
Should be fixed as of Bun v0.1.12 but possibly before that. Please comment if it isn't.
